packages/plugin-auto-tracking-browser/README.md

```diff
@@ -0,0 +1,76 @@
+<p align="center">
+  <a href="https://amplitude.com" target="_blank" align="center">
+    <img src="https://static.amplitude.com/lightning/46c85bfd91905de8047f1ee65c7c93d6fa9ee6ea/static/media/amplitude-logo-with-text.4fb9e463.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# @amplitude/plugin-auto-tracking-browser
```

Comment:
can we get this name approved by growth and dx teams (or other stakeholders)?

Reply:
Good call! This plugin is for internal use at present, will definitely need more discussion/approval about the naming and interface if we want to make it ready to public users.

Reply:
are we publishing to npm? it'll still be public if so. and we can't rename packages in npm